src/main/java/com/photolocos/enterprise/dao/PhotoDAOStub.java

```diff
@@ -0,0 +1,89 @@
+package com.photolocos.enterprise.dao;
+
+import com.photolocos.enterprise.dto.Photo;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+/**
+ * @author Paramjyot Sandhu (sandhups@mail.uc.edu)
+ *
+ * PhotoDAO class to handle Photo related operations
+ */
+@Component
+public class PhotoDAOStub implements IPhotoDAO {
+
+    Map<Integer, Photo> photos = new HashMap<>();
+
+    @Override
+    public boolean createEntry(Photo photo) throws Exception {
+        photos.put(photo.getPhotoId(), photo);
+        if (photos.containsKey(photo.getPhotoId())) return true;
+        return false;
+    }
+
+    @Override
+    public Set<Photo> fetchByTag(String[] tags) {
+        List<Photo> photosCollection = new ArrayList<>(photos.values());
+        Set<Photo> matchedPhotos = new HashSet<>();
+
+        for (Photo photo : photosCollection) {
+            Set<String> tagsSet = new HashSet<>(Arrays.asList(photo.getTags()));
+            for (String tag: tags) {
+                if (tagsSet.contains(tag)) {
+                    matchedPhotos.add(photo);
+                    continue;
```

Comment:
Should this be break?
break == break out of this loop entirely
continue == continue the loop with the next iteration.
Since this is the last line, continue isn't doing anything.  I suspect you meant to use break.